### PR TITLE
Added KojiTag model and it's relationships

### PR DESCRIPTION
**Pull request by yashvardhannanavati**

### CODE REVIEW

**Comment by yashvardhannanavati** [comment]
Assuming we will only store active tags

File: purview/models/koji.py [hunk 1]
```diff
@@ -34,3 +35,10 @@ class KojiTask(StructuredNode):
     priority = IntegerProperty()
     arch = StringProperty(required=True)
     method = StringProperty(required=True)
+
+
+class KojiTag(StructuredNode):
+    id_ = UniqueIdProperty(db_property='id')
+    creator = RelationshipTo('.user.User', 'CREATED_BY')
```
Comment (mprahl) [inline purview/models/koji.py]: Can you use `owners` here for consistency with our other models? Then the relationship name in Neo4j should be `OWNED_BY`.

Comment (mprahl) [reply purview/models/koji.py]: On second thought, maybe we don't need this data at all.

Comment (yashvardhannanavati) [reply purview/models/koji.py]: Sure, I can remove that relationship

Comment (mprahl) [reply purview/models/koji.py]: Okay, if you do can you remove the inverse as well?

Comment (yashvardhannanavati) [reply purview/models/koji.py]: Absolutely!, now please enjoy your sunday night. you can review this tomorrow :)

File: purview/models/koji.py [hunk 2]
```diff
@@ -34,3 +35,9 @@ class KojiTask(StructuredNode):
     priority = IntegerProperty()
     arch = StringProperty(required=True)
     method = StringProperty(required=True)
+
+
+class KojiTag(StructuredNode):
+    id_ = UniqueIdProperty(db_property='id')
+    builds = RelationshipTo('KojiBuild', 'TAGGED_TO')
```
Comment (mprahl) [inline purview/models/koji.py]: Based on our discussion, this should likely be called `CONTAINED_BY` for more generic relationship names.

File: purview/models/koji.py [hunk 3]
```diff
@@ -9,6 +9,7 @@ class KojiBuild(StructuredNode):
     id_ = UniqueIdProperty(db_property='id')
     tasks = RelationshipTo('KojiTask', 'TRIGGERS')
     owner = RelationshipTo('.user.User', 'OWNED_BY')
+    tags = RelationshipTo('KojiTag', 'TAGGED_BY')
```
Comment (mprahl) [inline purview/models/koji.py]: Based on our discussion, this should likely be called `CONTAINED_BY` for more generic relationship names.

**Comment by mprahl** [comment]
:+1:

**Pull request merged**